funnel/static/js/scripts.js

```diff
@@ -67,6 +67,10 @@ function activate_widgets() {
       }, 300);
     });
   });
+
+  $(document).on('select2:open', function () {
+    document.querySelector('.select2-search__field').focus();
```

Comment:
Upgrade to jquery 3.6 select2 autofocus isn't working. This is to fix that problem.
https://github.com/select2/select2/issues/5993

Reply:
This should be a comment in the code, for anyone who has to maintain this against a future jQuery/Select2 release.